### PR TITLE
[streamload](2pc) Fix 2pc stream load txn in cloud mode

### PR DESCRIPTION
**Pull request by gavinchou**
Abort load txn with label only should be forwarded to FE master to handle due to lack of db id.

### CODE REVIEW

**Comment by gavinchou** [comment]
run buildall

**Review by dataroaring — DISMISSED** [review]
LGTM

File: be/src/cloud/cloud_meta_mgr.cpp [hunk 1]
```diff
@@ -839,8 +839,12 @@ Status CloudMetaMgr::abort_txn(const StreamLoadContext& ctx) {
     if (ctx.db_id > 0 && !ctx.label.empty()) {
         req.set_db_id(ctx.db_id);
         req.set_label(ctx.label);
-    } else {
+    } else if (ctx.txn_id > 0) {
         req.set_txn_id(ctx.txn_id);
+    } else {
+        LOG(WARNING) << "failed abort txn, with illegal input, db_id=" << ctx.db_id
+                     << " txn_id=" << ctx.txn_id << " label=" << ctx.label;
+        return Status::InternalError("failed to abort txn");
```
Comment (liaoxin01) [inline be/src/cloud/cloud_meta_mgr.cpp:847]: No need to print stack？

**Comment by gavinchou** [comment]
run buidlall

**Comment by gavinchou** [comment]
run buildall

**Review by Yukang-Lian — APPROVED** [review]

**Review by dataroaring — APPROVED** [review]
LGTM

**Pull request merged**